docs/modules/technical-guide/pages/common-concepts/security.adoc

```diff
@@ -392,6 +392,10 @@ public static class ApiIdSignatureFilterContributor implements IServletFilterCon
 }
 ----
 
+IMPORTANT: A signed `IId` may contain special characters such as `#` or `;`. When such an `IId` is used as an URL parameter it needs to be encoded (see https://datatracker.ietf.org/doc/html/rfc3986#section-2[RFC 3986]).
```

Comment:
* According to the homepage, the datatracker is the _"day-to-day front-end to the IETF database for people who work on IETF standards."_. Maybe better link to https://www.ietf.org/rfc/rfc3986.html#section-2 instead?
* Explicitly mention that parameters always have to be encoded in the application, e.g. using encodeURIComponent() in JavaScript or IOutility.encode() in Java. This is technically implied in this first sentence, but because the rest of the paragraph focuses on infrastructure components, this might not be very clear to the reader.